src/cc_catalog_airflow/dags/sync_commoncrawl_workflow.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.python_operator import PythonOperator
+
+from commoncrawl_s3_syncer import SyncImageProviders
+from util.operator_util import get_log_operator
+
+
+DAG_DEFAULT_ARGS = {
+    'owner': 'data-eng-admin',
+    'depends_on_past': False,
+    'start_date': datetime(2020, 1, 15),
+    'email_on_retry': False,
+    'retries': 3,
+    'retry_delay': timedelta(days=1)
+}
+
+DAG_ID = "commoncrawl_workflow"
+
+
+def get_runner_operator(dag):
+    return PythonOperator(
+        task_id="sync_commoncrawl",
+        python_callable=SyncImageProviders.main,
+        depends_on_past=False,
+        dag=dag
+    )
```

Comment:
Unfortunately, this won't work.  It runs into an issue because there's an `ArgumentParser` defined in the main function of `SyncImageProviders` (just figured this out myself, or I would have mentioned it in the last review).  For this one, please use a `BashOperator` as in the original `monthly_workflow.py`.  Sorry for the confusion.

Reply:
Yes I had noticed that it checks for CC-MAIN- along with date. But wasn't sure of how to add the arguement ,I should have confirmed it , sorry .
Cant we add optional arguement in the PythonOperator to pass it.

Reply:
@mathemancer 
so cant we try this ``` op_args = "CC-MAIN-{ ds }" ```

Reply:
We could add that functionality at a later date.  The problem is that the `{ ds }` won't actually be the correct date.  When called without an argument, the script will just find the most recent date for which data is available.  I don't want to modify the script too much (or at all) right now, it's important to get it working in its current state first.

Reply:
ok , I didn't see any argument being passed in the monthly_workflow script to the Bash Operator , 
```s3SyncTask = BashOperator(task_id='Sync_Common_Crawl_Image_Data',                bash_command='python{0}/dags/commoncrawl_s3_syncer/SyncImageProviders.py'.format(airflowHome),dag=dag)```. 
I didn;t understand how that is passing the validation in ```SyncImageProviders```.